### ISSUE TITLE
Bump to rails 7 in not pushed to rubygems

### ISSUE THREAD

**Issue by K4sku** [richardvenneman/trestle-active_storage#121]
Hi,
Last push to rubygems is for 3.0.1 from 2020 and does not work for rails 7.

How about version bump 3.0.2 and push to Rubygems?

**Comment by marcus-pereira-repassa** [comment]
is it done?

**Comment by K4sku** [comment]
No. Nothing so far. You can use my fork if you want, it has fixes for has_many_attached

**Comment by marcus-pereira-repassa** [comment]
Thanks @K4sku .

**Comment by oriolblanc** [comment]
Is there a plan to update the gem? Or should we still use the @K4sku 's fork?

**Comment by fabiantorresm** [comment]
What is the route to fork @K4sku ?

**Comment by K4sku** [comment]
@fabiantorresm Do you mean the URL to my fork? It's https://github.com/K4sku/trestle-active_storage

**Comment by fabiantorresm** [comment]
thank you so much @K4sku

**Comment by fabiantorresm** [comment]
@K4sku how do I install it? can you tell me please. I used gem 'trestle-active_storage' and it still shows an error, I think it must have a path from your fork but I don't know

**Comment by fabiantorresm** [comment]
> @K4sku how do I install it? can you tell me please. I used gem 'trestle-active_storage' and it still shows an error, I think it must have a path from your fork but I don't know

Sorry, I already did it, it looked like this: 
`gem 'trestle-active_storage', github: 'richardvenneman/trestle-active_storage', branch: 'master'`

**Comment by K4sku** [comment]
In gemfile add 

git: "https://github.com/K4sku/trestle-active_storage.git", ref: "282e570"

To this gem name

**Comment by fabiantorresm** [comment]
Thank you very much! @K4sku

**Comment by DaddyWozBucks** [comment]
Unfortunately I have this specified using `Gem::Specification` so github urls arent going to work. Any chance this will happen sometime soon?
